mcs/class/System/Mono.UnityTls/UnityTls.cs

```diff
@@ -0,0 +1,249 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace Mono.Unity
+{
+    using UInt8 = Byte;
+    using Int8 = Byte;
+
+    [StructLayout (LayoutKind.Sequential)]
+    internal struct size_t
```

Comment:
Not blocking, but does this `size_t` provide enough value over just using `IntPtr`?

Reply:
It's mostly about the implicit cast to and from int. Since size_t is used for buffer sizes / array length, it would require quite a few explicit casts